### ISSUE TITLE
Use docker for aurblobs deployment / create DOCKERFILE

### ISSUE THREAD

**Issue by thilobillerbeck** [mweinelt/aurblobs#58]
It would be nice to deploy aurblobs inside a docker container for easy maintenance. Thus allowing for easy replacement and attachment to crontab and webservers. The container itself reaches the host's docker socket by a passed through socket. For configuration I have no idea yet.

**Comment by thilobillerbeck** [comment]
*I already forked and took up development of this feature, but I just wanted to mention it here*

**Comment by mweinelt** [comment]
This means docker would need to run privileged, correct?

**Comment by thilobillerbeck** [comment]
Yes, that's the only problem. But it is more an optional way to run aurblobs than the go-to solution. If someone wants to run aurblobs in unprivileged docker, then this person could still use a "native" deployment.

**Comment by thilobillerbeck** [comment]
I personally like this solution for "docker only" systems and linux distributions that are "designed" to dockerize everything.
